workflows/theiaprok/wf_theiaprok_ont.wdl

```diff
@@ -2,7 +2,7 @@ version 1.0
 
 import "../utilities/wf_read_QC_trim_ont.wdl" as read_qc
 import "../utilities/wf_merlin_magic.wdl" as merlin_magic_workflow
-#import "../../tasks/assembly/task_dragonflye.wdl" as dragonflye_task
+import "../../tasks/assembly/task_dragonflye.wdl" as dragonflye_task
```

Comment:
I like the notation of adding "_task" to the end of tasks to differentiate them from sub wf's. Might be worth including in a round of reformatting at some point.

Reply:
it also helps differentiate the task file name from the particular task within the task file.

Reply:
I can go through and reformat when I merge 👍